wlog/wrappedlog/wrapped1log/context.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2018 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wrapped1log
+
+import (
+	"context"
+)
+
+type wrapped1LogContextKeyType string
+
+const contextKey = wrapped1LogContextKeyType(TypeValue)
+
+// WithLogger returns a copy of the provided context with the provided Logger included as a value. This operation will
+// replace any logger that was previously set on the context (along with all parameters that may have been set on the
+// logger).
+func WithLogger(ctx context.Context, logger Logger) context.Context {
+	// Set _all_ the underlying logger types
```

Comment:
What does this comment mean? Does this code do something beyond just replace the logger on the specific context?

Reply:
Never mind, see updated comment that this will not be needed

Reply:
Is this still needed? Per my last comment, thought you had a previous comment that stated that this wouldn't need to be set here